SharedSources/Coordinators/AppCoordinator.swift

```diff
@@ -0,0 +1,32 @@
+//
```

Comment:
I'm not sure about this here btw. I wanted to go first with a coordinator pattern but later decided that it might not be the right choice. What could I use instead ? @dcordero any ideas ?